xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -1028,7 +1051,15 @@ This member is an explicit interface member implementation. It can be used only
         <param name="index">The position at which to insert the group.</param>
         <param name="item">The group to insert.</param>
         <summary>Inserts a <see cref="T:System.Text.RegularExpressions.Group" /> into the collection. Calling this method always throws <see cref="T:System.NotSupportedException" />.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method is not supported. This is a read-only collection.
+
+          ]]></format>
+        </remarks>
```

Comment:
Move to summary.
```suggestion
        <remarks>To be added.</remarks>
```

Reply:
I updated the summary to say "Calling this method always throws <see cref="T:System.NotSupportedException" />.". We can leave the remarks as they are now right? Even if it doesn't show up in intellisense, it helps to find the methods that are read-only